middleware/logger.go

```diff
@@ -46,7 +46,8 @@ type (
 		// Example "${remote_ip} ${status}"
 		//
 		// Optional. Default value DefaultLoggerConfig.Format.
-		Format string `yaml:"format"`
+		Format                string `yaml:"format"`
+		CustomTimestampFormat string `yaml:"custom_timestamp_format"`
```

Comment:
- Let's call this property as `TimeFormat`
- Put a line break between `Format` and `TimeFormat` properties and add a comment
- Include this tag (`time_custom`) in the list of tags at the end of time tags

Reply:
Thank you for your kind advise :)
I have fixed what you pointed out, so pls review!